test/viewing.jl

```diff
@@ -1,79 +1,135 @@
+    g = CartesianGrid{T}(10, 10)
     v = view(g, 1:3)
     @test unview(v) == (g, 1:3)
     @test unview(g) == (g, 1:100)
 
-    g = CartesianGrid{T}(10,10)
-    b = Box(P2(1,1), P2(5,5))
+    g = CartesianGrid{T}(10, 10)
+    b = Box(P2(1, 1), P2(5, 5))
     v = view(g, b)
-    @test v == CartesianGrid(P2(1,1), P2(5,5), dims=(4,4))
+    @test v == CartesianGrid(P2(1, 1), P2(5, 5), dims=(4, 4))
 
     p = PointSet(collect(vertices(g)))
     v = view(p, b)
-    @test centroid(v, 1) == P2(1,1)
-    @test centroid(v, nelements(v)) == P2(5,5)
+    @test centroid(v, 1) == P2(1, 1)
+    @test centroid(v, nelements(v)) == P2(5, 5)
 
-    g = CartesianGrid{T}(10,10)
+    g = CartesianGrid{T}(10, 10)
     p = PointSet(collect(vertices(g)))
-    b = Ball(P2(0,0), T(2))
+    b = Ball(P2(0, 0), T(2))
     v = view(g, b)
     @test nelements(v) == 1
     @test v[1] == g[1]
     v = view(p, b)
     @test nelements(v) == 6
-    @test coordinates.(v) == V2[(0,0),(1,0),(2,0),(0,1),(1,1),(0,2)]
+    @test coordinates.(v) == V2[(0, 0), (1, 0), (2, 0), (0, 1), (1, 1), (0, 2)]
   end
 
   @testset "Data" begin
     dummydata(domain, table) = DummyData(domain, Dict(paramdim(domain) => table))
     dummymeta(domain, table) = meshdata(domain, Dict(paramdim(domain) => table))
 
     for dummy in [dummydata, dummymeta]
-      g = CartesianGrid{T}(10,10)
+      g = CartesianGrid{T}(10, 10)
       t = (a=1:100, b=1:100)
       d = dummy(g, t)
       v = view(d, 1:3)
       @test unview(v) == (d, 1:3)
       @test unview(d) == (d, 1:100)
 
-      g = CartesianGrid{T}(10,10)
+      g = CartesianGrid{T}(10, 10)
       t = (a=1:100, b=1:100)
       d = dummy(g, t)
-      b = Box(P2(1,1), P2(5,5))
+      b = Box(P2(1, 1), P2(5, 5))
       v = view(d, b)
-      @test domain(v) == CartesianGrid(P2(1,1), P2(5,5), dims=(4,4))
-      @test Tables.columntable(values(v)) == (a=[12,13,14,15,22,23,24,25,32,33,34,35,42,43,44,45],
-                                              b=[12,13,14,15,22,23,24,25,32,33,34,35,42,43,44,45])
+      @test domain(v) == CartesianGrid(P2(1, 1), P2(5, 5), dims=(4, 4))
+      @test Tables.columntable(values(v)) == (
+        a=[12, 13, 14, 15, 22, 23, 24, 25, 32, 33, 34, 35, 42, 43, 44, 45],
+        b=[12, 13, 14, 15, 22, 23, 24, 25, 32, 33, 34, 35, 42, 43, 44, 45]
+      )
 
       p = PointSet(collect(vertices(g)))
       d = dummy(p, t)
       v = view(d, b)
       dd = domain(v)
-      @test centroid(dd, 1) == P2(1,1)
-      @test centroid(dd, nelements(dd)) == P2(5,5)
+      @test centroid(dd, 1) == P2(1, 1)
+      @test centroid(dd, nelements(dd)) == P2(5, 5)
       tt = Tables.columntable(values(v))
-      @test tt == (a=[13,14,15,16,17,24,25,26,27,28,35,36,37,38,39,46,47,48,49,50,57,58,59,60,61],
-                   b=[13,14,15,16,17,24,25,26,27,28,35,36,37,38,39,46,47,48,49,50,57,58,59,60,61])
-      
-      g = CartesianGrid{T}(250,250)
-      t = (a=rand(250*250), b=rand(250*250))
+      @test tt == (
+        a=[
+          13,
```

Comment:
I think these columns were incorrectly edited by the robot. Can they live in a single line?